providers/src/airflow/providers/apache/druid/transfers/hive_to_druid.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         self.sql = sql
         self.druid_datasource = druid_datasource
         self.ts_dim = ts_dim
-        self.intervals = intervals or ["{{ ds }}/{{ tomorrow_ds }}"]
+        self.intervals = intervals or ["{{ ds }}"]
```

Comment:
Hmm… we need to make this work across versions. Maybe something like this:

```suggestion
        self.intervals = intervals or ["{{ ds }}/{{ logical_date.add_days(1) | ds }}"]
```

Reply:
changed it